app/models/medium.rb

```diff
@@ -162,38 +162,22 @@ class Medium < ApplicationRecord
 
   # these are all the sorts of food(=projects) we currently serve
   def self.sort_enum
-    ["Kaviar", "Erdbeere", "Sesam", "Kiwi", "Nuesse", "Script", "Question", "Quiz", "Reste",
-     "Remark", "RandomQuiz"]
+    ["LessonMaterial", "WorkedExample", "Quiz", "Repetition", "Erdbeere",
+     "Exercise", "Script", "Question", "Remark", "Miscellaneous", "RandomQuiz"]
```

Comment:
Was it intentional to change the order here? Probably not a problem though but might be worth to check out again.

Reply:
Yes, this was intentional. I did not like the old ordering. I do not see any problems arising from this.